pages/common/docker-images.md

```diff
@@ -0,0 +1,20 @@
+# docker images
+
+> Manage Docker images.
+> More information: <https://docs.docker.com/engine/reference/commandline/images/>.
+
+- List all Docker images:
+
+`docker images`
+
+- List all Docker images with intermediate included:
+
+`docker images -a`
+
+- List all Docker images numeric IDs:
```

Comment:
We should try to match the `-q` with the description. Let us change it to - "List the output in quiet mode (only numeric IDs)"

Reply:
good suggestion 👍